todo_app/data/trello_items.py

```diff
@@ -75,11 +75,16 @@ def add_card(title):
     todo_list = get_idlist("To Do")
 
     # set up the card details as required
-    details = "name=" + title
-    details = details + "&pos=top"
-    details = details + "&desc=This is a new card"
-
-    response = requests.post(f"https://api.trello.com/1/cards?idList={todo_list}&key={api_key}&token={api_token}&{details}")
+    body = {
+        'name': title,
+        'pos': 'top',
+        'desc': 'This is a new card',
+        'key': api_key,
+        'token': api_token,
```

Comment:
Just be careful I think you missed the function calls here so it doesn't manage to create cards as is:
```suggestion
        'key': api_key(),
        'token': api_token(),
```